### PR TITLE
Adds various interesting badges.

### PR DESCRIPTION
**Pull request by Eremiell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # GoIV, Android Project for Pokémon GO.
 
 [![Travis Build Status](https://img.shields.io/travis/farkam135/GoIV/master.svg?maxAge=2592000 "Travis Build Status")](https://travis-ci.org/farkam135/GoIV)
+[![Github All Releases](https://img.shields.io/github/downloads/farkam135/GoIV/total.svg?maxAge=2592000)](https://github.com/farkam135/GoIV/releases)
+[![GitHub release](https://img.shields.io/github/release/farkam135/GoIV.svg?maxAge=2592000)](https://github.com/farkam135/GoIV/releases)
```
Comment (Blaisorblade) [inline README.md]: So we have these two release badges below (from @DJCrapsody), and they're linked to sth. else. I guess we want some consistency?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
 # GoIV, Android Project for Pokémon GO.
 
 [![Travis Build Status](https://img.shields.io/travis/farkam135/GoIV/master.svg?maxAge=2592000 "Travis Build Status")](https://travis-ci.org/farkam135/GoIV)
+[![Downloads](https://img.shields.io/github/downloads/farkam135/GoIV/total.svg?maxAge=2592000 "Downloads")](https://github.com/farkam135/GoIV/releases/latest)
+[![Changelog](https://img.shields.io/github/release/farkam135/GoIV.svg?maxAge=2592000 "Changelog")](CHANGELOG.md)
+[![Commits](https://img.shields.io/github/commits-since/farkam135/GoIV/3.1.0.svg?maxAge=2592000 "Commits")](https://github.com/farkam135/GoIV/compare/3.1.0...master)
```
Comment (DJCrapsody) [inline README.md]: @Blaisorblade @nahojjjen @sarav thoughts on commits badge?

File: README.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
 # GoIV, Android Project for Pokémon GO.
 
 [![Travis Build Status](https://img.shields.io/travis/farkam135/GoIV/master.svg?maxAge=2592000 "Travis Build Status")](https://travis-ci.org/farkam135/GoIV)
+[![Downloads](https://img.shields.io/github/downloads/farkam135/GoIV/total.svg?maxAge=2592000 "Downloads")](https://github.com/farkam135/GoIV/releases/latest)
+[![Changelog](https://img.shields.io/github/release/farkam135/GoIV.svg?maxAge=2592000 "Changelog")](CHANGELOG.md)
```
Comment (DJCrapsody) [inline README.md]: Would prefer Downloads and Changelog badges to be on next line. (commits to stay on the same line)

### Latest Release: [Downloads](https://github.com/farkam135/GoIV/releases/latest)
### Changelog: [Changelog](CHANGELOG.md)

**Comment by sarav** [comment]
If this doesn't add more rules or steps to making commits then I really don't have any opinion on this. If it'll add steps or rules to commits or coding, then a definite no please.

File: README.md (hunk 2 above)
Comment (Blaisorblade) [inline README.md]: We don't know yet how to do it without hardcoding a release name, it's not that important.

**Comment by Blaisorblade** [comment]
IIUC this was merged by the cherry-picks from #474, except the debated commit badge. So I guess I should close this for now.

**Pull request closed without merging**